src/components/reposition-relative-to-target.js

```diff
@@ -0,0 +1,28 @@
+AFRAME.registerComponent("reposition-relative-to-target", {
```

Comment:
Will do. Also _I think_ they're both wrong if `this.el.object3D` has a parent with an offset, since then setting the (local) `position` of the `object` using a world space vector will be wrong.